core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/theme/HedvigTheme.kt

```diff
@@ -37,3 +37,28 @@ fun HedvigTheme(
     }
   }
 }
+
+@Composable
+fun HedvigTheme(
+  darkTheme: Boolean = isSystemInDarkTheme(),
+  flipBackgroundAndSurface: Boolean,
```

Comment:
Could we rename this to something like `useNewColorScheme`?

Reply:
Yes absolutely!